### ISSUE TITLE
DOC: update installation page

### ISSUE THREAD

**Issue by lucascolley** [numpy/numpy.org#811]
We just overhauled the SciPy installation page at https://scipy.org/install/, including:
- creating a separate beginner guide for those who may not be comfortable with using a terminal or happy to learn to use a package manager
- recommending project-based workflows to users who do not consider themselves beginners (`uv` and `pixi`)
- modernising instructions for installing with `pip` or `conda`

You may wish to do similar things for https://numpy.org/install/. Feel free to copy anything I wrote on the SciPy page!

**Comment by mattip** [comment]
It might be nice to set this up as a tab-based page with a tab for each installation method. This would emphasize that they are equivalent alternatives.

**Comment by Musharraffaijaz** [comment]
I tried working on the same issues, but if you could kindly review the PR, Fixed the minor errors which hindered the development server #814 after which I have faced issues with the index page not avaliable, and shows **_Page not found_** 
![image](https://github.com/user-attachments/assets/a173830a-ca0a-4f87-8e72-36f871947f6d)

**Comment by Musharraffaijaz** [comment]
Hi, 
@lucascolley @mattip I'm working on a PR that would involve making some lengthier commits to the installation instructions in the `install.md` file. 
If thats fine, I'd be happy to create the PR for your review.

Thanks,

**Comment by Musharraffaijaz** [comment]
@mattip what about this? Since in markdowns, there is no option for Tabs and we can get tabs in MDX instead, so instead of a tab based navigation, how about the tabular one?  

![Image](https://github.com/user-attachments/assets/8c28a5e4-634c-4fbb-80dc-49533f238671)

**Comment by mattip** [comment]
If you want you could try something like [this](https://yairgadelov.me/simple-tab-widget-for-hugo-blog/) for tabs. I think the tabular (table) view wastes too much valuable space.

**Comment by Musharraffaijaz** [comment]
The PR  **[updated the installation page with tab based navigation #608](https://github.com/scipy/scipy.org/pull/608)**  resolves this issues

**Comment by lucascolley** [comment]
Hi @Musharraffaijaz - it looks like your PR affects scipy.org. This issue was about potential updates to numpy.org.

I'll have a look at your PR to see whether it looks like an improvement for SciPy! But I don't think it would close this issue.

**Comment by jarrodmillman** [comment]
Here is how you use tabs with the theme: https://theme.scientific-python.org/user_guide/web-components/#tabs

**Comment by jarrodmillman** [comment]
Here's more information: https://theme.scientific-python.org/shortcodes/#tabs

**Comment by Musharraffaijaz** [comment]
> We just overhauled the SciPy installation page at https://scipy.org/install/, including:
> 
> * creating a separate beginner guide for those who may not be comfortable with using a terminal or happy to learn to use a package manager
> * recommending project-based workflows to users who do not consider themselves beginners (`uv` and `pixi`)
> * modernising instructions for installing with `pip` or `conda`
> 
> You may wish to do similar things for https://numpy.org/install/. Feel free to copy anything I wrote on the SciPy page!

@lucascolley I have used the same tabs approach which Mattip suggested, and what I understood from the issue is that there are 5 installation methods for scipy, which I tried to separate using these tabs. The tabs are I had used are similiar to what @jarrodmillman has suggested.

**Comment by Musharraffaijaz** [comment]
@lucascolley @mattip what could be the improvements to the https://numpy.org/install/ page? I mean do we have to try the similiar tab-based navigation as we did in the scipy/install page?

**Comment by mattip** [comment]
It would be nice to refactor the NumPy pages to be much like the SciPy page after that PR lands.

**Comment by Musharraffaijaz** [comment]
> It would be nice to refactor the NumPy pages to be much like the SciPy page after that PR lands.

Thank you for the reply, I will begin working on the task immediately.  While that PR has been approved, I would greatly appreciate it if you could conduct a final review to facilitate its merge.  Once that is complete, I will commence work on the NumPy integration.

**Comment by mattip** [comment]
I'm not sure, but I think I prefer the layout 1:1 from the https://scipy.org/install/ page instead of what we have. What do others think?

**Comment by melissawm** [comment]
I agree, the scipy page looks awesome and the contents should be very similar so I'd vote to follow that.

**Comment by Musharraffaijaz** [comment]
@mattip @melissawm thank you for your review, could you please check the PR **_["Update the numpy installation page with tab based navigation"](https://github.com/numpy/numpy.org/pull/831)_** which tries to update the installation page

**Comment by Musharraffaijaz** [comment]
@mattip @melissawm thank you for your review, could you please check the PR "**_[Update the numpy installation page with tab based navigation](https://github.com/numpy/numpy.org/pull/832)_**" which tries to update the Numpy installation page.

**Comment by Musharraffaijaz** [comment]
@lucascolley @mattip could you please review this PR #832

**Comment by melissawm** [comment]
Hi @Musharraffaijaz - sorry for the delay, it looks like we are all a bit busy but we'll make sure to come back to this PR when we can. Thanks for the patience!

**Comment by Musharraffaijaz** [comment]
@melissawm sure, sorry for any inconvenience caused.

**Comment by Musharraffaijaz** [comment]
I guess this issue is now resolved, @mattip do you consider any other project or issue where I could contribute to?

**Comment by mattip** [comment]
Thanks for fixing this. As for more issues, you could look at the ones in numpy and scipy that are marked "DOC".

**Comment by Musharraffaijaz** [comment]
Alright, thanks.
